### ISSUE TITLE
Could not find assembly 'System.Runtime.InteropServices.PInvoke'

### ISSUE THREAD

**Issue by russkyc** [skuill/LyricsScraperNET#8]
#### Bug
Build fails after updating to v1.1.0 in both .net6 and .net7 avalonia and wpf projects after updating to v1.1.0

#### Error:
```
Microsoft.WinFX.targets(211, 9): [MC1000] Unknown build error, 'Could not find assembly 'System.Runtime.InteropServices.PInvoke, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. Either explicitly load this assembly using a method such as LoadFromAssemblyPath() or use a MetadataAssemblyResolver that returns a valid assembly.' 
```

#### Environment
- Project Type: WPF/Avalonia
- SDK: .NET 7, .NET 6

**Comment by skuill** [comment]
Hi @russkyc !
Thank you for finding this issue.
The problem is reproduced. The reason is the addition of a new [MusixmatchClientLib](https://github.com/Eimaen/MusixmatchClientLib) library to the project, which removes the assembly:
![image](https://github.com/skuill/LyricsScraperNET/assets/19705282/bd74c56e-00aa-4230-924c-928a4d677169)
Adding the **System.Runtime.InteropServices** or **System.Runtime.InteropServices.RuntimeInformation** nuget packages did not resolve the issue..
I am working on a fix. As soon as I can fix it, I will let you know. Temporarily please stay on the previous version :)

**Comment by skuill** [comment]
Hi @russkyc!
The problem should be fixed in release 1.1.1. Can you please check that the project build with the new version?

**Comment by russkyc** [comment]
Just tested, @skuill Project now compiles and works normally, thanks for resolving the issue!
